gitlab/Caddyfile-socket

```diff
@@ -0,0 +1,24 @@
+https://dev.sth.se.k0nsl.org {
+
+    log git.access.log {
+        rotate_size 50 # Rotate after 50 MB
+        rotate_age  80 # Keep rotated files for 90 days	
+        rotate_keep 15 # Keep at most 20 log files
+    }
+    errors {
+        404 /opt/gitlab/embedded/service/gitlab-rails/public/404.html
+        422 /opt/gitlab/embedded/service/gitlab-rails/public/422.html
+        500 /opt/gitlab/embedded/service/gitlab-rails/public/500.html
+        502 /opt/gitlab/embedded/service/gitlab-rails/public/502.html
+    }
+
+    proxy / unix:/home/git/gitlab/tmp/sockets/gitlab.socket {
```

Comment:
To be perfectly honest with you, I don't know for a certainty whether that is the case or not--if I am not mistaken the socket path should be definable in `/etc/gitlab/gitlab.rb`. But if GitLab was provisioned automatically via some type of turnkey solution in a pre-baked LXC container, then the socket path can always be found by checking `/home/git/gitlab/tmp/sockets`.